### PR TITLE
WE-164 style side navigation menu

### PR DESCRIPTION
**Pull request by logansparlin**
## What Changed
- Adds styling and expanding functionality to the side navigation

## How to verify
- Run locally or visit the deploy link to verify the styling and functionality are correct

### CODE REVIEW

**Review by EvanYoung-SP — CHANGES_REQUESTED** [review]
Just a few changes, but great work!

**Review by jonambas — COMMENTED** [review]
not sure if you saw, but there are some cypress screenshots in this PR

File: components/site/momentumNavigation.tsx [hunk 1]
```diff
@@ -1,41 +1,178 @@
...
   link: string;
   items?: this[];
+  level?: number;
 }
 
 export type MomentumNavigationProps = {
   data?: MomentumNavigationItemProps[];
 };
 
+const StyledLink = styled(Box)<BoxProps & { $active?: boolean }>`
```
[the remark this answers is not in the document]
Comment (logansparlin) [reply components/site/momentumNavigation.tsx:22]: That way styled components doesn't pass active in as an html attribute

**Review by EvanYoung-SP — APPROVED** [review]
LGTM!

**Pull request merged**